examples/greeter-client-eio/dune

```
@@ -0,0 +1,3 @@
+(executable
+ (name greeter_client_eio)
+ (libraries grpc grpc-eio ocaml-protoc eio greeter h2 h2-eio))
```

```suggestion
 (libraries grpc grpc-eio ocaml-protoc eio_main greeter h2 h2-eio))
```
This should depend on `eio_main`